src/overlays/actors/ovl_Bg_Spot16_Bombstone/z_bg_spot16_bombstone.c

```diff
@@ -269,8 +270,8 @@ void BgSpot16Bombstone_Init(Actor* thisx, PlayState* play) {
 
 #if DEBUG_FEATURES
         default:
-            PRINTF("Error : arg_data おかしいな(%s %d)(arg_data 0x%04x)\n", "../z_bg_spot16_bombstone.c", 668,
-                   this->actor.params);
+            PRINTF(T("Error : arg_data おかしいな", "Error : arg_data is strange") "(%s %d)(arg_data 0x%04x)\n",
```

Comment:
Seems like a similar phrase as to the `The main mode is wrong`. Feel like wrong would work here as well.

Reply:
yeah they are the same word ("okashii"). Basic googling says weird/strange is probably the better translation, but maybe "wrong" makes more sense in this specific context.

Can of course be cleaned up in a later pass from someone who isnt google translate. will merge the PR soon.